libs/platform/src/lib/components/form/radio-group/radio-group.component.ts

```diff
@@ -57,12 +57,6 @@ export class RadioGroupComponent extends CollectionBaseInput implements AfterVie
         super.setValue(newValue);
     }
 
-    /**
```

Comment:
a breaking change?

Reply:
have kept this now, did not remove it. Not a breaking change. earlier i had moved it to base class.

Reply:
not a breaking change. so marking it resolved.